eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -43,15 +43,12 @@ jobs:
       - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
         parameters:
           AgentImage: $(OSVmImage)
-
```

Comment:
Nit: IMO, I do like the extra line break for clarity, but not a big deal.

Reply:
When I revert the revert I'll keep the breaks in.